build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
 		classpath "org.ajoberstar:gradle-git-publish:${VER_GRADLE_GIT}"
 		classpath "ch.raffael.pegdown-doclet:pegdown-doclet:${VER_PEGDOWN_DOCLET}"
 		classpath "gradle.plugin.com.github.spotbugs:spotbugs-gradle-plugin:${VER_SPOTBUGS_PLUGIN}"
+		classpath "com.google.guava:guava:27.1-jre" // Force newer version for spotbugs
```

Comment:
Note that this should hopefully not be necessary anymore after Gradle 5.6 is released, which has a lot of work to properly not expose these dependencies on its classpath